### ISSUE TITLE
Module '"../../node_modules/@casl/react"' has no exported member 'BoundCanProps'.  TS2305

### ISSUE THREAD

**Issue by hckhanh** [stalniy/casl#419]
I got an error when I try to upgrade @casl/ability and @casl/react to the latest version 

```bash
Module '"../../node_modules/@casl/react"' has no exported member 'BoundCanProps'.  TS2305

    1 | import { Ability } from "@casl/ability";
  > 2 | import { BoundCanProps, createCanBoundTo } from "@casl/react";
      |          ^
    3 | import { Fade } from "@material-ui/core";
    4 | import { PropsWithChildren, ReactElement } from "react";
    5 | import ability from "../utils/ability";
```

**Comment by stalniy** [comment]
Yes, my bad. Give me few minutes to fix this. This is a configuration issue during build.

**Comment by stalniy** [comment]
just published `@casl/react@2.2.1` with a fix

**Comment by hckhanh** [comment]
Thank you @stalniy <3
